src/tests/Plot/Guid.cs

```diff
@@ -13,15 +13,15 @@ class Guid
         public void Test_Guids_AreUnique()
         {
             int reps = 100;
-            var guids = new List<string>();
+            var guids = new HashSet<string>();
```

Comment:
Does it really matter?
`Distinct` also uses hash, and if it slower then not many times.
100 is a small number...

Reply:
No, it doesn't really matter. It saves a little bit of time on my machine but it's mostly inconsequential.